RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -399,54 +430,64 @@ namespace mkfit {
       const auto ntracks = tracks.size();
 
       std::vector<float> ctheta(ntracks);
+      std::multimap<int, int> theHitMap;
```

Comment:
```suggestion
      std::multimap<int, int> hitMap;
```
`the` prefix is commonly used for data member names; it will be more clear here without it